alphadia/transferlearning/train.py

```diff
@@ -896,3 +897,166 @@ def finetune_charge(self, psm_df: pd.DataFrame) -> pd.DataFrame:
         metrics = test_metric_manager.get_stats()
 
         return metrics
+
+    def _test_ccs(
+        self,
+        epoch: int,
+        epoch_loss: float,
+        test_df: pd.DataFrame,
+        metric_accumulator: MetricManager,
+        data_split: str,
+    ) -> bool:
+        """
+        Test the CCS model using the PSM dataframe and accumulate both the training loss and test metrics.
+
+        Parameters
+        ----------
+        epoch : int
+            The current epoch number.
+        epoch_loss : float
+            The train loss value of the current epoch.
+        test_df : pd.DataFrame
+            The PSM dataframe.
+        metric_accumulator : MetricManager
+            The metric manager object.
+        data_split : str
+            The dataset label to test on. e.g. "validation", "train"
+        Returns
+        -------
+        bool
+            Whether to continue training or not based on the early stopping criteria.
+        """
+        continue_training = True
+        if epoch % self.settings["test_interval"] == 0:
```

Comment:
this would save on indent:
```python
if epoch % self.settings["test_interval"] != 0:
    return continue_training

self.ccs_model.model.eval()
...
```